### ISSUE TITLE
Download page not making 1.6 version for macOS available.

### ISSUE THREAD

**Issue by mike-silva** [balena-io/etcher#3604]
The download page doesn't make the .dmg for the 1.6 version available for macOS.

**Comment by lurch** [comment]
It's not just macOS - **all** the download links on https://www.balena.io/etcher/ are still for 1.5.122

**Comment by thundron** [comment]
yes, we had an issue when trying to publish the links. it will be solved with next version which I'm working on
